libs/components/forms/src/lib/modules/checkbox/checkbox-group.component.ts

```diff
@@ -0,0 +1,69 @@
+import { CommonModule } from '@angular/common';
+import {
+  Component,
+  HostBinding,
+  Input,
+  booleanAttribute,
+  inject,
+} from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { SkyIdService } from '@skyux/core';
+
+import { SKY_FORM_ERRORS_ENABLED } from '../form-error/form-errors-enabled-token';
+import { SkyFormErrorsModule } from '../form-error/form-errors.module';
+
+/**
+ * Organizes checkboxes into a group.
+ */
+@Component({
+  selector: 'sky-checkbox-group',
+  templateUrl: './checkbox-group.component.html',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+    `,
+  ],
+  standalone: true,
+  imports: [CommonModule, SkyFormErrorsModule],
+  providers: [{ provide: SKY_FORM_ERRORS_ENABLED, useValue: true }],
+})
+export class SkyCheckboxGroupComponent {
+  /**
+   * The text to display as the checkbox group's label.
+   * @preview
+   */
+  @Input({ required: true })
+  public labelText!: string;
+
+  /**
+   * Indicates whether to hide the `labelText`.
+   * @preview
+   */
+  @Input({ transform: booleanAttribute })
+  public labelHidden = false;
+
+  /**
+   * Whether the checkbox group is stacked on another form component. When specified, the appropriate
+   * vertical spacing is automatically added to the checkbox group.
+   */
+  @Input({ transform: booleanAttribute })
+  public set stacked(value: boolean) {
+    this.cssClass = value ? 'sky-margin-stacked-lg' : '';
+  }
+
+  /**
+   * The form group that contains the group of checkboxes.
+   * @preview
+   */
+  @Input({ required: true })
+  public formGroup!: FormGroup;
+
+  @HostBinding('class')
+  public cssClass = '';
```

Comment:
So I was also concerned this would be the case but when I tested it out, it still respects any consumer-given classes, even when they're changed. It also respects the various form status classes the `formGroup` puts on it.

Reply:
Interesting.... when I googled it - it appeared it would do the opposite. If it doesn't remove others - then I guess ship it 😆